source/fuzz/transformation_add_dead_continue.cpp

```diff
@@ -58,12 +58,6 @@ bool TransformationAddDeadContinue::IsApplicable(
     return false;
   }
 
-  assert(bb_from != nullptr &&
-         "We should have found a block if this line of code is reached.");
-  assert(
-      bb_from->id() == message_.from_block() &&
-      "The id of the block we found should match the source id for the break.");
```

Comment:
I removed these since we have already done all the checks for `nullptr` above.

Reply:
Again, I would keep these defensive assertions.

Reply:
Fixed in 2a0b8d1.